src/pages/Login/Login.jsx

```diff
@@ -1,5 +1,211 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
 
 export default function Login() {
-  return <div></div>;
+  const [emailValue, setEmailValue] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+  const [isEmailValid, setIsEmailValid] = useState('');
+  const [isPasswordValid, setIsPasswordValid] = useState('');
+
+  const controlValue = event => {
+    if (event.target.name === 'inputEmail') {
+      setEmailValue(event.target.value);
+    } else if (event.target.name === 'inputPassword') {
+      setPasswordValue(event.target.value);
+    }
+  };
+
+  const allValueCheck = () => {
+    const reg_email = /^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/;
+
+    const reg_pwd =
+      /^(?=.*[A-Za-z])(?=.*\d)(?=.*[@$!%#?&])[A-Za-z\d@$!%*#?&]{8,16}$/;
+
+    setIsEmailValid(reg_email.test(emailValue));
+    console.log('이메일 정규식 확인', isEmailValid);
+
+    setIsPasswordValid(reg_pwd.test(passwordValue));
+
+    console.log('비밀번호 정규식 확인', isPasswordValid);
+  };
+
+  const goToMain = e => {
+    e.preventDefault();
+
+    if (!isEmailValid && !isPasswordValid) {
+      return;
+    }
```

Comment:
```suggestion
    if (!isEmailValid && !isPasswordValid) return; // early return
```
- early return 활용 잘하셨네요!
- (return 문이 한 줄인 경우) 중괄호 생략은 해도 되고 안 해도 되는 부분입니다. 
- 저는 개인적으로 한 줄인 경우에는 생략을 하는 편이지만, 코드가 여러 줄로 추가되는 경우, 함께 작업하는 경우를 고려하여 무조건 중괄호를 추가해주는 개발자도 있습니다. 참고만 해주세요 :)

Reply:
네 참고하겠습니다 !